orderer/common/multichannel/registrar.go

```diff
@@ -96,6 +100,7 @@ func NewRegistrar(
 		config:             config,
 		chains:             make(map[string]*ChainSupport),
 		followers:          make(map[string]*follower.Chain),
+		pendingRemoval:     make(map[string]bool),
```

Comment:
why not map[string]struct ?

Reply:
@yacovm that has been suggested by Yoav in a follow-up https://github.com/hyperledger/fabric/pull/2027#discussion_r516476013